style.css

```diff
@@ -1,533 +1,627 @@
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+  -webkit-box-align: baseline;
+  -ms-flex-align: baseline;
+  align-items: baseline;
+  margin-right: 24px;
+}
+
+.languages a {
+  text-decoration: none;
+  color: #000;
+}
+
+.language {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  background-color: #f7f7f9;
+  padding: 10px;
+  margin-right: 30px;
+  margin-bottom: 0;
+}
+
+.language-text {
+  margin-left: 12px;
+  -ms-flex-item-align: center;
+  -ms-grid-row-align: center;
+  align-self: center;
+}
+
+.language img {
+  width: 48px;
+}
+
+.language h4 {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+}
+
+.frameworks {
+  margin-top: 30px;
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+.frameworks h3 {
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+}
+
+.frameworks .vector {
+  cursor: pointer;
+  width: 20px;
+  height: 11px;
+}
+
+.framework-list {
+  margin-bottom: 10px;
+  padding: 14px;
+  border-bottom: 1px solid #dfe1e6;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+}
+
+.button:hover {
+  background-color: #6070ff;
+  color: #fff;
+}
+
+.button:focus {
+  border: 0.0625rem solid #2230d2;
+  background-color: #2230d2;
+  color: #fff;
+}
+
+.button-enabled {
+  color: #396df2;
+  border: 0.0625rem solid #6070ff;
+}
+
+/* footer section( contact me form) */
+
+footer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  background: url(./images/contact\ form\ shapes-mobile1.svg) no-repeat;
+  background-color: #6070ff;
+  background-attachment: scroll;
+  background-size: contain;
+  height: 100vh;
+  border-top-left-radius: 70px;
+  background-position: right;
+}
+
+.form-contact {
+  display: flex;
+  flex-direction: column;
+  row-gap: 27px;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+}
+
+.head-contact {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-size: 40px;
+  color: white;
+  text-align: center;
+  margin-bottom: 0;
+}
+
+.form-contact > ul {
+  width: 100%;
+  list-style: none;
+}
+
+.form-ul {
+  padding: 0;
+  margin: 0;
+}
+
+.contactme > p {
+  color: white;
+  font-family: "Poppins", sans-serif;
+  font-size: 20px;
+  text-align: center;
+  width: 80%;
+}
+
+.contactme {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  justify-items: center;
+  padding: 0 25vw;
+}
+
+.input-form {
+  border-radius: 7px;
+  width: 100%;
+  height: 48px;
+  margin: 5px 0;
+  outline: none;
+  border: 1px solid #cfd8dc;
+  padding-inline-start: 5px;
+}

```

Comment:
- [ ] Kindly adhere to the project requirement because as I can see you are not using CSS properly. which makes your design `inconsistent` and looks unprofessional. Please have a look at the below-attached requirement image with the design comparison table and my friendly suggestion to you is to always try to make your design as close as possible. 

### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)

### Design Comparison
<table>
<tr>
<th>Your Design</th>
<th>Figma Design</th>
</tr>
<tr>
<td>

![image](https://github.com/Fentahunn/contact_form_final/assets/66476329/2deb9bba-5a05-4cfe-9089-b244d1e4ba0f)

</td>
<td>

![image](https://github.com/Fentahunn/contact_form_final/assets/66476329/05e99c50-d534-44b8-b0f7-2887602923b9)

</td>
</tr>
</table>